pyilc/wavelets.py

```diff
@@ -119,6 +146,98 @@ def plot_wavelets(self, filename='example_wavelet_plot', log_or_lin='log'):
         plt.grid(alpha=0.5)
         plt.savefig(filename+log_or_lin+'.pdf')
 
+class scale_info(object):
+    ## It might be better to let this inherit from the wavelets class, and just use one scale_info object instead of both wavelets
+    ## and scale_info - possibly a future edit
+    def __init__(self,wv,info,):
+        ##########################
+        # criterion to determine which frequency maps to use for each wavelet filter scale
+        # require multipole ell_F where wavelet filter F(ell_F) = wavelet_beam_criterion (on its decreasing side)
+        #   to be less than the multipole ell_B where the beam B(ell_B) = wavelet_beam_criterion
+        # note that this assumes monotonicity of the beam
+        # and assumes filter function has a decreasing side, which is generally not true for the smallest-scale wavelet filter
```

Comment:
I suppose the smallest-scale needlet filter for the cosine-needlet case indeed does have a decreasing side!